wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1347,23 +1347,23 @@ def keyboard_shortcuts_dialog(context):
                     f"{KEYS.MOD} + {KEYS.SHIFT} + z" if is_mac else f"{KEYS.MOD} + y",
                 ),
             ],
-            ("actions-model", _("Actions")): [
+            ("actions", _("Actions")): [
                 (_("Save changes"), f"{KEYS.MOD} + s"),
                 (_("Preview"), f"{KEYS.MOD} + p"),
-                (_("Toggle sidebar"), "["),
-                (_("Toggle minimap"), "]"),
                 (_("Add or show comments"), f"{KEYS.CTRL} + {KEYS.ALT} + m")
                 if comments_enabled
                 else None,
             ],
-            ("rich-text-content", _("Text content")): [
-                (_("Insert or edit a link"), f"{KEYS.MOD} + k")
```

Comment:
Don't forget to re-add "Insert or edit a link" to the "Text formatting" section :)